configurations/q6_pipeline_config.json

```diff
@@ -0,0 +1,104 @@
+{
+  "RapidProDomain": "textit.in",
+  "RapidProTokenFileURL": "gs://avf-credentials/imaqal-text-it-token.txt",
+  "RapidProTestContactUUIDs": [
+    "fad44f22-235a-4821-9780-dfef09076fe1",
+    "3903ad8e-284e-48a0-881b-89c0b06b9334",
+    "45939459-e4d5-4ad6-8650-3e7134b9a035"
+  ],
+  "RecoveryCSVURLs": [
+    "gs://avf-project-datasets/2019/IMAQAL/recovered_hormuud_drama_s01e06_de_identified.csv",
+    "gs://avf-project-datasets/2019/IMAQAL/recovered_hormuud_magazine_15_de_identified.csv",
+    "gs://avf-project-datasets/2019/IMAQAL/recovered_hormuud_magazine_15_wednesday_de_identified.csv"
+  ],
+  "PhoneNumberUuidTable": {
+    "FirebaseCredentialsFileURL": "gs://avf-credentials/avf-id-infrastructure-firebase-adminsdk-6xps8-b9173f2bfd.json",
+    "TableName": "IMAQAL"
+  },
+  "ActivationFlowNames": [
```

Comment:
Thank you @as2388. We might need to add this to the full_pipeline_config.json too.

Reply:
We will, but I was hesitant to do so until we needed to. For a future PR I think